src/system/MailerModule/Controller/AdminController.php

```diff
@@ -13,85 +13,181 @@
 
 namespace Zikula\MailerModule\Controller;
 
-use Symfony\Component\Security\Core\Exception\AccessDeniedException;
-use Zikula_View;
-use SecurityUtil;
-use FormUtil;
-use Zikula\MailerModule\Form\Handler\ModifyConfigHandler;
-use Zikula\MailerModule\Form\Handler\TestConfigHandler;
-use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpFoundation\RedirectResponse;
-use Symfony\Component\Routing\RouterInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Zikula\Core\Controller\AbstractController;
+use Zikula\ThemeModule\Engine\Annotation\Theme;
 
 /**
+ * Class AdminController
+ * @package Zikula\MailerModule\Controller
  * @Route("/admin")
  */
-class AdminController extends \Zikula_AbstractController
+class AdminController extends AbstractController
```

Comment:
would prefer to rename controllers based on the object they manage. In this case maybe `ConfigController`